cohort-evaluator-hapi-fhir/pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <image.tag>us.icr.io/vpc-dev-cohort-rns/cohort-evaluator-spark:a-p3.4.3-tekton-20210907202747-a57c643820323a5c76ddb8392aa64c3eab27ba45</image.tag>
-        <config.volume>cohort-config</config.volume>
+        <config.bucket>cohort-config</config.bucket>
         <data.bucket>kwasny-test</data.bucket>
         <data.secret>spark-cos-secret</data.secret>
         <job.name>simple-job</job.name>
```

Comment:
I think I incorrectly copied these from the Spark POM when I created this one.
You deleted the properties from `cohort-spark-evaluator/pom.xml`, probably should here as well.

Reply:
_resolved_ in 960a596